docs/content/en/contribute/docs/source-file-structure/_index.md

```diff
@@ -27,6 +27,8 @@ The meaning of these fields is:
 
 * **title** -- title displayed for the section or file
 * **description** -- subtext displayed for the section or subsection
+* **icon** -- An optional field specifying an icon associated with the section
+* **layout** -- The layout template to be used for rendering the section
```

Comment:
```suggestion
* **icon** -- an optional field specifying an icon associated with the section
* **layout** -- the layout template to be used for rendering the section
```
please be consistent with capitalization

Reply:
Actually, only these two items used an article before the description so the structure itself was inconsistent.  The articles aren't required so I deleted them